src/app/views/people/people.component.html

```diff
@@ -9,39 +9,40 @@
     </div>
   </div> -->
 
-  <div class="d-flex mt-5" style="margin-left: 25vw;width: 100vw;">
-    <h1 class="float-start">{{getTitleFromType(type)}} Directory</h1>
-  </div>
+  <div class="d-flex mt-5 " style="width: 100vw;">
```

Comment:
Explain this

Reply:
Where is the explanation ??

Reply:
Margin Left was Overwriting the usual 100vw, to take it beyond. Hence Overflow-x was required to curb that. Since I removed the Overflow-x case, I had to manage the overflow, so made the complete View Port size as 100vw.